app/views.py

```diff
@@ -5,12 +5,13 @@
 from flask import render_template, request, jsonify, flash, redirect, url_for
 from flask_login import current_user, login_user, logout_user, login_required
 from app import app, db, mail
-from app.models import User, Question, Option, Answer, Response, UserGroup, Group, Quiz, Thread, Post, Proficiency
+from app.models import User, Question, Option, Response, Group, Thread, Post, Proficiency, Quiz
 from app.forms import *
-from app.questions import get_question_options, submit_response, get_student_cat, get_response_answer
+from app.questions import get_question_options, submit_response, get_student_cat, get_response_answer, get_question_quiz
```

Comment:
If we are going to import everything why don't we just `from x import *`

Reply:
Because we are not importing everything.